packages/calcite-components/src/components/autocomplete/autocomplete.scss

```diff
@@ -0,0 +1,65 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-autocomplete-background-color: Specifies the background color of the component.
+ * @prop --calcite-autocomplete-text-color: Specifies the text color of the component.
+ */
```

Comment:
Does this need a `placeholder-text-color` prop?

Reply:
@macandcheese I think all these need to be added right? https://github.com/Esri/calcite-design-system/pull/10820